src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ReleaseFileServiceTests.cs

```diff
@@ -548,11 +459,26 @@ public async Task ZipFilesToStream_FiltersInvalidFileTypes()
 
             var publicBlobStorageService = new Mock<IPublicBlobStorageService>(MockBehavior.Strict);
 
+            var captureRequest = new CaptureZipDownloadRequest
```

Comment:
The other option here (and one that I used in Public API) is to set analytics enabled to false by default, so you don't have to add additional analytics-related expectations everywhere that might otherwise make non-related tests noisy.

Then I had a set of tests specifically for testing when it is on.

Don't know if this is any use to you or if you're happy with where you've added new expectations in here?

Reply:
Yup that's correct.